### PR TITLE
Build system updates

### PR DESCRIPTION
**Pull request by gmorph**
Added navio and navio2 binaries to be built for AntennaTracker
Added bebop as binary to be built for Copter.

### CODE REVIEW

File: Tools/scripts/build_binaries.sh [hunk 1]
```diff
@@ -267,6 +267,26 @@ build_arducopter() {
...
+            echo "Failed checkout of ArduCopter $b $tag"
+            error_count=$((error_count+1))
+            continue
+        }
+        skip_board $b && continue
+        echo "Building ArduCopter $b binaries"
+        ddir=$binaries/Copter/$hdate/$b
+        skip_build $tag $ddir && continue
+        make clean || continue
+        make $b -j4 || {
```
Comment (lucasdemarchi) [inline Tools/scripts/build_binaries.sh]: Make is deprecated for Linux boards and doesn't work anymore for some of them

Comment (rmackay9) [inline Tools/scripts/build_binaries.sh]: @lucasdemarchi, @guludo, any idea how we can add the bebop2 and navio builds?

Comment (lucasdemarchi) [inline Tools/scripts/build_binaries.sh]: just like we build them in Tools/scripts/build_ci.sh

Comment (OXINARF) [inline Tools/scripts/build_binaries.sh]: @rmackay9 Is that urgent? I'm thinking about reworking the *build_binaries.sh* file but I don't have an exact timeline (max two weeks I hope).

**Comment by rmackay9** [comment]
@OXINARF, it's not super urgent so it can wait a couple of weeks, thanks!

**Comment by rmackay9** [comment]
I'll close this one 'cuz I don't think it's getting in without some significant changes

**Pull request closed without merging**